src/components/browse/sidebar/sidebar.tsx

```diff
@@ -28,6 +34,8 @@ const Sidebar = (props: sidebarProps) => {
     });
     props.setFilterDict([]);
     props.setSearchKeyword("")
+    props.setPage(0)
+    navigate(`?p=1`)
```

Comment:
Same. I thought p=0 was what we wanted for displaying Page 1/first page.

Reply:
p starts with 1
default page is either 
/browse
or
/browse/?p=1

its never p=0

Reply:
I thought we were doing p=0 as the starting page. Is this not something that can be done? It's much easier and solves the previous comments.

Reply:
That creates a new set of problems, resetting the pagination start view from 0 instead of 1. The logical 0th page is hidden from the user and it is cleaner to start with p=1.

Reply:
Changed behaviour to my suggestion. Resolved.